generate.py

```diff
@@ -1,8 +1,10 @@
-import os
+from __future__ import print_function

```

Comment:
I wasn't sure which version of Python your repo was for (2 or 3) so I added this because of the difference between Python 2's `print` statement and Python 3's `print` function.

```python
Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print('hello', end='!\n')
hello!

Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print('hello', end='!\n')
  File "<stdin>", line 1
    print('hello', end='!\n')
                      ^
SyntaxError: invalid syntax
>>> from __future__ import print_function
>>> print('hello', end='!\n')
hello!

```
Adding the brackets around a Python 2 `print` statement is usually just ignored but if one of the function arguments was used it could cause errors.

This import is not absolutely necessary, I only put it there to try and prevent future errors in case the script was run using Python 2 (and used `print` function specific arguments).